models/etude.py

```diff
@@ -0,0 +1,13 @@
+import mongoengine as me
+
+class Etude(me.Document):
+    number = me.IntField(required=True)
+    organisme = me.StringField(required=True)
+    name = me.StringField(required=True)
+    id_follower_quality = me.StringField(required=True)
+    id_follower_study = me.StringField(required=True)
+    description = me.StringField(required=True)
```

Comment:
Same as above for organisme, id_follower_quality, id_follower_study
`organisme= ReferenceField('Organisme')`
`follower_quality= ReferenceField('User')`
`follower_study= ReferenceField('User')`